SolarSharp.Interpreter/Security/CrossPlatformPathCanonicalizer.cs

```diff
@@ -293,149 +232,13 @@ private Result<bool, PathSecurityViolation> ValidateLinuxPath(string path)
             return Result.Success<bool, PathSecurityViolation>(true);
         }
 
-        private string ResolveRealPath(string path)
```

Comment:
Doesn't do any thing.  And the better solution here (for netcore atleast) is LinkTarget in FileInfo (and there are other methods that integrate with it).  which **is** crossplatform.

More generally though I'm not convinced this is worth doing.  Users can't create symlinks in windows without admin and it's pretty easy for us to block in linux.  So you would have had to give explicit permission to a path with a symlink but then not want someone to use it?

With the concept of a VFS symlinks truly don't matter anyways other than we would want to replicate that, so I think that's likely the solution?  It's not like you can really **escape** from a symlink to execute something in it's parent directory anyways.